src/insights_client/tests/test_metrics.py

```diff
@@ -37,19 +37,19 @@ def factory(**config):
         config = u"""[server]
 hostname = cert-api.access.redhat.com
 port = 443
-proxy_hostname = %s
-proxy_port = %s
-proxy_user = %s
-proxy_password = %s
+%s
+%s
+%s
+%s
 
 [rhsm]
 repo_ca_cert =
 consumerCertDir =
 """ % (
-            config.get("proxy_hostname", ""),
-            config.get("proxy_port", ""),
-            config.get("proxy_user", ""),
-            config.get("proxy_password", ""),
+            "proxy_hostname = %s" % config["proxy_hostname"] if "proxy_hostname" in config else "",
+            "proxy_port = %s" % config["proxy_port"] if "proxy_port" in config else "",
+            "proxy_user = %s" % config["proxy_user"] if "proxy_user" in config else "",
+            "proxy_password = %s" % config["proxy_password"] if "proxy_password" in config else "",
```

Comment:
This too can be simplified.

```suggestion
""" % tuple(
            "%s = %s" % (key, config[key]) if key in config else ""
            for key in ("proxy_hostname", "proxy_port", "proxy_user", "proxy_password")
      )
```